internal/tests/cli/boundary/_target_host_sources.bash

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+load _authorized_actions
+
+function add_target_host_sources_sources() {
+    for i in "${@:2}"; do
+        hostSource+="-host-source $i "
+    done
+
+    boundary targets add-host-sources -id $1 $hostSource
+}
+
+function remove_target_host_sources_sources() {
+    for i in "${@:2}"; do
+        hostSource+="-host-source $i "
+    done
+
+    boundary targets remove-host-sources -id $1 $hostSource
+}
+
+function set_target_host_sources_sources() {
+    for i in "${@:2}"; do
+        hostSource+="-host-source $i "
+    done
+
+    boundary targets set-host-sources -id $1 $hostSource
+}
+
+function validate_host_sources() {
```

Comment:
It was a little difficult to quickly understand what this method is trying to do

A lot of the methods in this test suite start off by defining what the input variables are, like...
```
local tid=$1
local hsid=$2
local format=$3
```
It might make sense to do that here?

Reply:
yeah, that's better. I'll set the format as a variable and group all the input variables at the top

Reply:
yeah, I like that. More descriptive